velox/functions/sparksql/aggregates/DecimalSumAggregate.h

```diff
@@ -23,7 +23,7 @@ namespace facebook::velox::functions::aggregate::sparksql {
 /// @tparam TInputType The raw input data type.
 /// @tparam TSumType The type of sum in the output of partial aggregation or the
 /// final output type of final aggregation.
-template <typename TInputType, typename TSumType>
+template <typename TInputType, typename TSumType, uint8_t ResultPrecision>
```

Comment:
Perhaps add some comments for the new template parameter.

Reply:
Sure, added.